usaspending_api/etl/management/commands/create_delta_table.py

```diff
@@ -31,43 +34,48 @@
         "custom_schema": "",
     },
     "recipient_lookup": {
-        "schema_sql_string": recipient_lookup_sql_string,
+        "model": RecipientLookup,
         "source_table": "recipient_lookup",
         "destination_database": "raw",
         "partition_column": "id",
         "partition_column_type": "numeric",
+        "schema_sql_string": recipient_lookup_sql_string,
```

Comment:
It may be too much of a refactor for this scope, but do you and @collinwr also think that the term `schema_sql_string` can be misleading? Should it be more explicit like `table_create_sql` or something along those lines. "Schema" is getting overloaded to mean a lot of things currently in this context.

Reply:
I generally agree, as that's the primary purpose of it and the schema is more laid out in the model/db. @collinwr mind if I change it to `table_create_sql`?